src/style.css

```diff
@@ -1 +1,43 @@
 /* add styles here */
+.html {
+  margin: 0;
+}
+
+/* #region stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  padding-right: 4px;
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1:nth-child(-n+1) {
```

Comment:
The selector `.stars--1:nth-child(-n+1)` is incorrect for targeting the first child. The `.stars--1` class is applied to the parent div, not the individual stars. Consider using `.stars--1 .stars__star:nth-child(-n+1)` to target the first star within the `.stars--1` div.